package.json

```
@@ -1,11 +1,18 @@
 {
   "name": "zksync-hardhat-template",
-  "version": "1.0.0",
-  "main": "index.js",
-  "author": "Antonio <aug@matterlabs.dev>",
+  "description": "A template for zkSync smart contracts development with Hardhat",
+  "private": true,
+  "author": "Matter Labs",
   "license": "MIT",
+  "repository": "https://github.com/matter-labs/zksync-hardhat-template.git",
+  "scripts": {
+    "deploy": "yarn hardhat deploy-zksync --script deploy.ts",
+    "interact": "yarn hardhat deploy-zksync --script interact.ts",
+    "test": "hardhat test --network hardhat"
```

nit: Should this specify `yarn`'s version of `hardhat` in case there's a global version?
```suggestion
    "test": "yarn hardhat test --network hardhat"
```